apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/SpringValueProcessor.java

```diff
@@ -33,7 +34,8 @@
  * @author github.com/zhegexiaohuozi  seimimaster@gmail.com
  * @since 2017/12/20.
  */
-public class SpringValueProcessor extends ApolloProcessor implements BeanFactoryPostProcessor, BeanFactoryAware {
+public class SpringValueProcessor extends ApolloProcessor implements BeanFactoryPostProcessor,
```

Comment:
There are many code format changes, please use the [apollo-code-style](https://github.com/ctripcorp/apollo/tree/master/apollo-buildtools/style) and remove these unnecessary changes.